add-chain/e2e_test.go

```diff
@@ -10,9 +10,9 @@ import (
 )
 
 const (
-	addressDir             = "../superchain/extra/addresses/sepolia/"
-	configDir              = "../superchain/configs/sepolia/"
-	genesisSystemConfigDir = "../superchain/extra/genesis-system-configs/sepolia/"
+	addressDir         = "../superchain/extra/addresses/sepolia/"
+	configDir          = "../superchain/configs/sepolia/"
+	validtionInputsDir = "../validation/genesis/validation-inputs"
```

Comment:
```suggestion
	validationInputsDir = "../validation/genesis/validation-inputs"
```

(needs usages to be fixed as well)